proposals/flow-vs-typescript.md

```diff
@@ -0,0 +1,69 @@
+# Flow vs TypeScript
+
+Flow and TypeScript are both fantastic ways to limit technical debt in your JavaScript projects, allowing you to, among other things, evolve your codebase with confidence.
+They are similar tools that focus on static types checking.
+
+# Flow
+
+## Pros:
+
+1. As you're using React, we might prefer Flow because it easily integrates with Babel and other infrastructure.
+
+2. Flow is known to be a simpler language than TypeScript due to it’s out of the box utility. A developer would have to learn TypeScript syntax e.g. Type Annotation and how to modify the code.
```

Comment:
Isn't this an equal point for both Flow and TypeScript? They both have their own type annotation syntax